universe/org.themoviedb/eval/scenarios.txt

```diff
@@ -41,3 +41,14 @@ A: I see Tom Cruise, Tom Cruise, and Tom Cruise Jr.
 U: \t sort(popularity desc of @org.themoviedb.actor());
 
 A: I see .*, .*, and .*.
+====
+# 7-filtered-actor-search
+U: \t @org.themoviedb.actor() filter id=~ "Scarlett Johansson";
+
+A: I see Scarlett Johansson.
+
+====
+# 8-filter-movie-by-actor
+U: \t @org.themoviedb.movie() filter contains~(actors, "Scarlett Johansson");
+
+A: I see Avengers: Endgame, The Avengers, and Avengers: Infinity War. (They're movies by Robert Downey Jr|Avengers: Endgame is a movie released in April 23, 2019, The Avengers is released in April 24, 2012, and Avengers: Infinity War is released in April 24, 2018).
```

Comment:
Given that Avengers is the "default" query term, it's probably a good idea to add a test with a different actor/actress.